sage-ios/SAGE/SAGE/AddAnnouncementController.swift

```diff
@@ -40,6 +40,7 @@ class AddAnnouncementController: UIViewController {
             let finalAnnouncement = self.addAnnouncementView.exportToAnnouncement()
             AdminOperations.createAnnouncement(finalAnnouncement, completion: { (announcement) -> Void in
                 self.navigationController!.popViewControllerAnimated(true)
+                NSNotificationCenter.defaultCenter().postNotificationName(NotificationConstants.addAnnouncementKey, object: finalAnnouncement)
```

Comment:
finalAnnouncement or the announcement passed in the completion block? probably better to use the anonuncement instead since it retains the correct id number